code/datums/status_effects/rogue/roguebuff.dm

```diff
@@ -230,3 +230,43 @@
 	. = ..()
 	to_chat(owner, span_warning("The rough floors slow my travels once again."))
 	REMOVE_TRAIT(owner, TRAIT_LONGSTRIDER, MAGIC_TRAIT)
+
+/atom/movable/screen/alert/status_effect/buff/healing
+	name = "Healing Miracle"
+	desc = "Divine intervention relieves me of my ailments."
+	icon_state = "buff"
+
+/datum/status_effect/buff/healing
+	id = "healing"
+	alert_type = /atom/movable/screen/alert/status_effect/buff/healing
+	duration = 10 SECONDS
+	var/healing_on_tick = 1
+
+/datum/status_effect/buff/healing/tick()
+	var/obj/effect/temp_visual/heal/H = new /obj/effect/temp_visual/heal(get_turf(owner))
+	H.color = "#FF0000"
+	owner.adjustBruteLoss(-healing_on_tick)
+	owner.adjustFireLoss(-healing_on_tick)
+	var/list/wCount = owner.get_wounds()
+	if(owner.blood_volume < BLOOD_VOLUME_NORMAL)
+		owner.blood_volume = min(owner.blood_volume+10, BLOOD_VOLUME_NORMAL)
+	if(wCount.len > 0)
+		owner.heal_wounds(healing_on_tick)
+		owner.update_damage_overlays()
+	owner.adjustBruteLoss(-healing_on_tick, 0)
```

Comment:
is it intentional that you're doubling up on brute+fire healing here? lines 248 and 249 already seem to do this which makes me think not

Reply:
Ack. My bad. Will fix when I get home.

Reply:
Ack. My bad. Will fix when I get home.